src/pages/ReimbursementAccount/EnableStep.js

```diff
@@ -1,56 +1,134 @@
+            throw new Error('Account not found in EnableStep');
+        }
+
+        const {icon, iconSize} = getBankIcon(account.additionalData.bankName);
+        const formattedBankAccountNumber = account.accountNumber
+            ? `${translate('paymentMethodList.accountLastFour')} ${
+                account.accountNumber.slice(-4)
+            }`
+            : '';
+        const bankName = account.addressName;
+        return (
+            <View style={[styles.flex1, styles.justifyContentBetween]}>
+                <HeaderWithCloseButton
+                    title={translate('workspace.common.bankAccount')}
+                    onCloseButtonPress={Navigation.dismissModal}
+                    shouldShowBackButton
+                    onBackButtonPress={() => Navigation.goBack()}
                 />
-                <Text style={[styles.mh5, styles.mb5]}>
-                    {translate('validationStep.reviewingInfo')}
-                    <TextLink
-                        onPress={() => {
-                            // There are two modals that must be dismissed before we can reveal the Concierge
-                            // chat underneath these screens
-                            Navigation.dismissModal();
-                            Navigation.dismissModal();
-                            navigateToConciergeChat();
-                        }}
+                <View style={[styles.flex1]}>
+                    <WorkspaceSection
+                        title={!isUsingExpensifyCard ? translate('workspace.bankAccount.oneMoreThing') : translate('workspace.bankAccount.allSet')}
+                        IconComponent={() => (!isUsingExpensifyCard ? <Icon src={ConciergeBlue} width={80} height={80} /> : <Image source={confettiPop} style={styles.confettiIcon} />)}
+                        menuItems={!isUsingExpensifyCard ? [{
+                            title: translate('workspace.bankAccount.chatWithConcierge'),
+                            icon: ChatBubble,
+                            onPress: () => {
+                                Navigation.dismissModal();
+                                navigateToConciergeChat();
+                            },
+                            shouldShowRightIcon: true,
+                        }] : []}
                     >
-                        {translate('common.here')}
-                    </TextLink>
-                    {translate('validationStep.forNextSteps')}
-                </Text>
+                        <MenuItem
+                            title={bankName}
+                            description={formattedBankAccountNumber}
+                            icon={icon}
+                            iconWidth={iconSize}
+                            iconHeight={iconSize}
+                            onPress={() => {}}
+                            wrapperStyle={{paddingHorizontal: 0, marginBottom: 12}}
```

Comment:
```suggestion
                            wrapperStyle={...styles.ph0, ...styles.mb3}
```

Or better yet, refactor `MenuItem` to take an object or array of styles in the `wrapperStyles` prop.